### PR TITLE
Allow `lts` for the LTS, and allow `pre` for the latest pre-release

### PR DESCRIPTION
**Pull request by KristofferC**
for now, this just hardcodes the LTS version in the source (similar to how juliaup does it) since the latest LTS is not available in `versions.json`. Since the LTS is updated so rarely this might be ok for now.

### CODE REVIEW

**Comment by SaschaMann** [comment]
> Since the LTS is updated so rarely this might be ok for now.

Yea, seems fine to require an action update for that once every few years.

---

For some reason the GHA logs aren't showing me the error properly except in the raw logs, it's

```
2024-04-15T12:15:06.8345144Z ##[error] found changed files after build. please 'npm run build && npm run format' and check in all changes
2024-04-15T12:15:06.8354990Z ##[error]Process completed with exit code 1.
```

That CI check still runs on Node16. #235 should fix it

**Comment by ViralBShah** [comment]
Can we get this merged?

**Comment by DilumAluthge** [comment]
Hmmm. PR Checks are still failing.

File: .github/workflows/example-builds.yml
Comment (DilumAluthge) [inline .github/workflows/example-builds.yml:22]: Should we also test `pre`, since that's added in this PR?

```suggestion
        julia-version: ['1.0.5', '1.2', '^1.5.0-beta1', '1', 'lts', 'pre']
```

File: __tests__/installer.test.ts
Comment (DilumAluthge) [inline __tests__/installer.test.ts:66]: @KristofferC Will this test will need to be updated every time there's a new pre-release?

**Comment by DilumAluthge** [comment]
We should decide on a case (uppercase vs lowercase) convention, and keep it consistent between this PR and #202.

Which do we prefer?
1. Lowercase: `lts`, `pre`, and `min`
2. Uppercase: `LTS`, `PRE`, and `

**Review by DilumAluthge — APPROVED** [review]
I think this is probably fine, but it might be good to get another review.

**Review by DilumAluthge — APPROVED** [review]

**Comment by DilumAluthge** [comment]
Phew, CI is all green now.

**Comment by ViralBShah** [comment]
Nice! Will `pre` install alphas, betas, rcs, etc? It is probably worth mentioning directly in the README.

**Comment by IanButterworth** [comment]
I'm working on testing this on an external repo to demo it, then will update the readme for review, before releasing 2.2.0

**Comment by IanButterworth** [comment]
https://github.com/IanButterworth/PowerMonitor.jl/actions/runs/9686654256/job/26729572283?pr=4

There are failures in the package on apple silicone, ignore those, but otherwise seems good

**Comment by IanButterworth** [comment]
@ViralBShah check the readme under https://github.com/julia-actions/setup-julia#examples

If that looks good then I can release v2.2.0

**Comment by ViralBShah** [comment]
I proposed https://github.com/julia-actions/setup-julia/pull/256

**Pull request merged**
